doc/source/basic-use.rst

```diff
@@ -1,153 +1,259 @@
+This basic jail is fully independent.
 
-``iocage create``
+To create a full jails with multiple clones, type:
 
-This will create a fully independent jail.
+:samp:`# iocage create -r 11.0-RELEASE -c 2`
 
-To create a lightweight jail (clone) run:
+The numeric value after the **-c** flag is used to designate the number
+of clone jails to create. In the above example, *two* jails are created,
+**one** *full* jail and **one** clone jail.
 
-``iocage create -c``
+A simple basejail is created with the **-b** flag:
 
-To create a base jail:
+:samp:`# iocage create -b -r 11.0-RELEASE`
 
-``iocage create -b``
+After designating the type of jail to create with the option flags,
+specific jail **properties** can also be set. For example:
 
-To create a jail and set its IP address and tag property run:
+:samp:`# iocage create -r 11.0-RELEASE tag=myjail boot=on`
 
-``iocage create -c tag=myjail ip4_addr="em0|10.1.1.10/24"``
+Creates a FreeBSD 11.0 jail with the custom tag *myjail* and sets the
+jail to start at system boot time.
 
-For more information please read iocage(8).
+More information about iocage jail properties is available in the
+iocage(8) FreeBSD manual page, which is accessed on a FreeBSD system by
+typing :command:`man iocage`.
 
-Listing jails
+.. index:: Listing Jails
+.. _Listing Jails:
+
+Listing Jails
 -------------
 
-To list all jails run:
+To list all jails:
+
+:samp:`# iocage list`
+
+To see all downloaded RELEASEs:
+
+:samp:`# iocage list -r`
+
+View available templates with:
 
-``iocage list``
+:samp:`# iocage list -t`
 
-To see all downloaded RELEASEs run:
+.. index:: Jail start stop restart
+.. _Start Stop Restart Jail:
 
-``iocage list -r``
+Start, Stop, or Restart a Jail
+------------------------------
 
-To see available templates run:
+Jails can be started, stopped, or restarted at any time with iocage. By
+default, new jails are in a *down* (stopped) state. To see the status of
+all jails, use :command:`iocage list` and read the **STATE** column.
 
-``iocage list -t``
+Use each jail's UUID or custom TAG to start, stop, or restart it. When
+using a jail's UUID, it is not required to type the full UUID. Partial
+UUIDs are acceptable, as long as the given characters are enough to
+identify the desired jail. Alternately, use **ALL** to apply the command
+to all created jails.
 
-Start, stop or restart a jail
------------------------------
+.. tip:: Partial UUIDs can also be supplied for any other iocage
+   operation or subcommand.
 
-To start or stop any jail on the system both the UUID or TAG can be used interchangeably.
-To simplify UUID handling iocage accepts a partial UUID too with any subcommand.
+.. index:: Jail Start
+.. _Startjail:
 
 Start
 +++++
 
-To start a jail tagged www01 simply run:
+To start a jail with the custom tag **www01**, type:
 
-``iocage start www01``
+:samp:`# iocage start www01`
 
-To start a jail with a full UUID run:
+A jail can also be started with a full UUID:
```

Comment:
This can be elaborated with (default naming scheme) or something similar. As that will remain our default, unless the user specifies `--name`